src/genlisp/generate.py

```diff
@@ -827,9 +827,10 @@ def generate_srv_from_spec(msg_context, spec, search_path, output_dir, package,
     "Generate code from .srv file"
     genmsg.msg_loader.load_depends(msg_context, spec, search_path)
     ext = '.srv'
-    srvs = [f[:-len(ext)] for f in os.listdir(os.path.dirname(path)) if f.endswith(ext)]
-    for s in srvs:
-        load_srv_from_file(msg_context, path, '%s/%s'%(package, s))
+    srv_path = os.path.dirname(path)
+    srvs = [f[:-len(ext)] for f in os.listdir(srv_path) if f.endswith(ext)]
```

Comment:
I very rarely use python, and cannot really detect code smells in it. This would how I would write the same line using ```msg_list```: ```srvs = msg_list(package, {package: [srv_path]}, ext)```
Does that look better?